### DISCUSSION TITLE
Codeforces: Çarpım Tablosu

### DISCUSSION THREAD

**Discussion by DevFloppy** [sonsuzus/sonsuzus#463] [Soru Cevap] [answered]
Petya konumsal gösterimleri inceliyor. Farklı köklere sahip gösterim sistemlerinde sayıları toplamayı ve çıkarmayı çoktan öğrendi ve daha karmaşık bir eyleme geçti — çarpma. Büyük sayıları çarpmak için çarpım tablosunu öğrenmek gerekir. Ne yazık ki, ikinci sınıfta öğrenciler yalnızca ondalıkların çarpım tablosunu öğreniyorlar (ve bazı öğrenciler bunu birinci sınıfta bile öğreniyor). Petya'nın k köklü gösterim sistemindeki sayılar için bir çarpım tablosu yapmasına yardım edin .

Giriş
İlk satır tek bir tam sayı k ( 2 ≤  k  ≤ 10 ) içerir — sistemin tabanı.

Çıktı
k tabanlı gösterim sisteminin çarpım tablosunu çıktı olarak verin . Tablo k  - 1 satır ve k  - 1 sütun içermelidir. i -inci satır ile j -inci sütunun kesiştiği noktadaki eleman, k tabanlı gösterim sistemindeki i ve j'nin çarpımına eşittir . Her satır sayılar arasında herhangi sayıda boşluk içerebilir (örneklerdeki ekstra boşluklar açıklık sağlamak için konulmuştur).

**Comment by DevFloppy** [comment, marked as answer]
```python
yazdır=[]
sayı=int(input)
if 2<=sayı<=10:
    i=int(0)
    while i<(sayı-1):
        yazdır.clear()
        t=int(0)
        while t<(sayı-1):
            yazdır.append((i+1)*(t+1))
            t+=1
        print(yazdır)
        i+=1
```

**Reply by sonsuzus** [reply]
kodunu kod satırları arasına aldım.

Örnek girdi ve çıktıları da eklersen soruya güzel olur.
